### PR TITLE
Allow Flexible Library Versions

### PR DESCRIPTION
**Pull request by schbetsy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,25 @@ coverage run manage.py test
...
+
+[Documentation](https://cfpb.github.io/owning-a-home-api/) for this repository is rendered via GitHub pages and [Swagger](https://swagger.io/docs/). They can be edited in the `docs/` directory, but to view or deploy them, you'll need to install the dependencies listed in the `docs_extras` section of `setup.py`:
+
+```
+pip install -e '.[docs]'
+```
+
+You can then preview your changes locally by running `mkdocs serve` and then reviewing <http://127.0.0.1:8000/>
+
+When your changes are ready, you can submit them as a normal pull request. After that, you can use this command to publish them:
```
Comment (chosak) [inline README.md]: Thanks for doing this. I noticed that [the current live docs](https://cfpb.github.io/owning-a-home-api/mysql/) are out of date. After we merge this PR, let's run the command to refresh the GH branch.

File: README.md [hunk 2]
```diff
@@ -88,6 +88,25 @@ coverage run manage.py test
 coverage report
 ```
 
+
+## API Docs
+
+[Documentation](https://cfpb.github.io/owning-a-home-api/) for this repository is rendered via GitHub pages and [Swagger](https://swagger.io/docs/). They can be edited in the `docs/` directory, but to view or deploy them, you'll need to install the dependencies listed in the `docs_extras` section of `setup.py`:
```
Comment (chosak) [inline README.md]: Swagger isn't used by the docs here; was this copied from [ccdb5-api](https://github.com/cfpb/ccdb5-api#api-docs)?

Comment (schbetsy) [reply README.md]: Oops. Yes, I did copy-paste this text. Fixed now.

File: README.md (hunk 1 above)
Comment (schbetsy) [reply README.md]: Good call. Will do.

Comment (schbetsy) [reply README.md]: Done 👍

Comment (chosak) [reply README.md]: Thanks!

**Pull request merged**